.travis.yml

```
@@ -0,0 +1,6 @@
+language: node_js
+node_js: "node"
+services: 
+  -mongodb
+env:
+  MONGODB_URI=mongodb://127.0.0.1:27017/users AUTH_SECRET=password
```

```suggestion
  MONGODB_URI=mongodb://127.0.0.1:27017/users
```
No secret necessary here as we are not dealing with tokens.